### ISSUE TITLE
Accessibility issue on top navigation

### ISSUE THREAD

**Issue by gsin11** [KhalisFoundation/sttm-web#1389]
**Describe the bug**
As a keyboard user, I cannot navigate through all the top navigation links

**To Reproduce**
1. Go to https://www.sikhitothemax.org
2. Start navigating through the keyboard
3. You will see focus never comes on the Sync dropdown.

**Expected behavior**
Sync dropdown/CTA should get keyboard focus so keyboard users can access it as well.

**Screenshots**
![image](https://user-images.githubusercontent.com/3865313/114378017-9190ac80-9b87-11eb-9be6-4b3cef41ac9c.png)

**Desktop (please complete the following information):**
 - OS: All
 - Browser: All
 - Version All

**Additional context**
*Suggestion: As per accessibility guidelines, the drop-down should be a `<button>` instead `<p>`. Since the button is an interactive element that will take keyboard focus automatically and will be readable by screen readers.*

**Comment by gsin11** [comment]
Code has been merged into the dev branch and already available on http://dev.sikhitothemax.org/

**Comment by gsin11** [comment]
reopening since we are waiting for this issue to be deployed on PROD.
